src/services/auth/auth_test.go

```diff
@@ -28,18 +30,33 @@ func TestLogin(t *testing.T) {
 			want: 200,
 		},
 	}
+
+	expetedLogin:="admin"
+	expetedPass:="admin"
+
 	data := url.Values{}
-	data.Add("login", "admin")
-	data.Add("password", "admin")
+	data.Add("login", expetedLogin)
+	data.Add("password", expetedPass)
 	originalDB := db
 	defer func() { db = originalDB }()
+	var err error
 	db, mock, err = sqlmock.New()
```

Comment:
Do you use db variables in your tests?

Reply:
@mshto is correct. Handler auth should know nothing about DB. If you need to mock function from models, mock it in models